spacy/cli/info.py

```diff
@@ -10,6 +9,7 @@
 from .download import get_model_filename, get_latest_version
 from .. import util
 from .. import about
+import importlib.metadata
```

Comment:
Sorry, can you move this up to where `pkg_resources` was instead of having with the relative imports?